specification/appendix/examples/examples.mdpp

```diff
@@ -0,0 +1,5 @@
+# Examples
+
+*This section is non-normative.*
+
+!INCLUDE "metadata_examples/metadata_examples.mdpp",1
```

Comment:
I'm going to follow the commitment utilization pr with some examples like this.  Can we standardize the path to: `appendix/examples/<category>/...` and remove the 2nd `examples`?

So: `/metadata_examples/` becomes `/metadata/`

Reply:
perhaps we call it column examples?

Reply:
Done